rake/spec/default/99-uninstall-system-packages_spec.rb

```diff
@@ -0,0 +1,6 @@
+# Check that StackStorm packages can be uninstalled without errors
+describe 'st2 packages uninstall test' do
+  describe command("$BASEDIR/scripts/remove_os_packages.sh #{spec[:package_list]}") do
```

Comment:
Don't need a script.

Can check the OS flavor in Serverspec/Rspec
I could find https://serverspec.org/advanced_tips.html#how-to-get-os-information

Reply:
Thanks - I was actually just looking into that :)